spec/controllers/downloads_controller_spec.rb

```diff
@@ -5,10 +5,10 @@
   describe "with a file" do
     let(:depositor) { FactoryGirl.find_or_create(:archivist) }
     let(:file) do
-      GenericFile.new.tap do |f|
+      GenericFile.create do |f|
         f.apply_depositor_metadata(depositor.user_key)
-        f.add_file(File.open(fixture_path + '/world.png'), 'content', 'world.png')
-        f.save!
+        f.label = 'world.png'
```

Comment:
We used to have a method that took care of setting the label and a default title, and I see the actor now does this itself in create_content. (And is "label" even a thing in Fedora 4?  I thought was a Fedora 3 built-in.)